### PR TITLE
#38: ejected text to a separate package

### PR DESCRIPTION
**Pull request by prowwid**
- updated dependencies in related packages
- updated package.json to have valid author and contributor properties
- moved RandomString to text package as it's related to text manipulation scope

----
[Refactoring: split core into individual packages](https://github.com/monumentjs/core/issues/38)

### CODE REVIEW

File: packages/benchmark/package.json [hunk 1]
```diff
@@ -4,6 +4,11 @@
   "version": "0.15.0",
   "description": "Benchmark utils",
   "keywords": [],
+  "author": {
+    "name": "Alex Chugaev",
+    "email": "achugaev93@gmail.com",
+    "url": "https://github.com/avchugaev"
+  },
```
Comment (avchugaev) [inline packages/benchmark/package.json]: you can add your name in this package too

File: packages/text/.npmignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+test
```
Comment (avchugaev) [inline packages/text/.npmignore]: `test` directory now renamed to `spec`, so we don't need this line

File: packages/text/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+[![Travis CI Status](https://img.shields.io/travis/monumentjs/core/master.svg?logo=travis)](https://travis-ci.org/monumentjs/core)
+[![Slack Chat](https://img.shields.io/badge/slack-chat-brightgreen.svg?logo=slack)](https://join.slack.com/t/monumentjs/shared_invite/enQtNDY1ODA1MTExMzQ4LTI0MjllODEwOTk5MjM0NGIwY2YwNzVjNDU3YjEwYzYwYTNjMmI0NjFkNmNjMDFlMjA1NzgzODk0NjcxZTc4NjM)
+[![npm version](https://badge.fury.io/js/%40monument%2Ftext.svg)](https://badge.fury.io/js/%40monument%2Ftext)
+![npm version](https://david-dm.org/monumentjs/text.svg)
+
+Provides text manipulation solutions
+
+Related links:
+
+- [API reference](https://monumentjs.github.io/package/collections)
```
Comment (avchugaev) [inline packages/text/README.md]: link should point to `text` package

File: packages/text/src/TemplateString.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { EMPTY_STRING } from './Strings';
-import { MissingKeyException } from '../exceptions/MissingKeyException';
+import { MissingKeyException, ToString } from '@monument/core';
```
Comment (avchugaev) [inline packages/text/src/TemplateString.ts]: I think we should find a better place for `MissingKeyException`. Probable it related to collections...

File: packages/text/README.md (hunk 3 above)
Comment (prowwid) [reply packages/text/README.md]: Good catch - I was sure I changed it:+1:
Fixed to point to text

File: packages/text/.npmignore (hunk 2 above)
Comment (prowwid) [reply packages/text/.npmignore]: removed it:ok_hand:

File: packages/benchmark/package.json (hunk 1 above)
Comment (prowwid) [reply packages/benchmark/package.json]: sure, added in the last commit:ok_hand:

File: packages/text/src/TemplateString.ts (hunk 4 above)
Comment (prowwid) [reply packages/text/src/TemplateString.ts]: To summarize what we discussed in chat: we'll eject all exceptions to a separate [ackage at the next step

**Pull request merged**